.scalafix.conf

```diff
@@ -0,0 +1,4 @@
+OrganizeImports {
+  groupedImports = Keep
+  coalesceToWildcardImportThreshold = 7
```

Comment:
I would use the organize imports to remove all unused imports.


```suggestion
  coalesceToWildcardImportThreshold = 7
  removeUnused = true
```

Reply:
Also it looks like the coalesce to wildcard is now breaking some stuff (importing both shapeless and cats `Id` into same scope)